.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Submit a bug report
+labels: "type-bug"
+---
+
+<!--
+  If you're new to Python and you're not sure whether what you're experiencing is a bug, the CPython issue tracker is not
+  the right place to seek help. Consider the following options instead:
+
+  - reading the Python tutorial: https://docs.python.org/3/tutorial/
+  - posting in the "Users" category on discuss.python.org: https://discuss.python.org/c/users/7
+  - emailing the Python-list mailing list: https://mail.python.org/mailman/listinfo/python-list
+  - searching our issue tracker (https://github.com/python/cpython/issues) to see if
+    your problem has already been reported
+-->
+
+**Bug report**
+
+A clear and concise description of what the bug is.
+Include a minimal, reproducible example (https://stackoverflow.com/help/minimal-reproducible-example), if possible.
+
+**Your environment**
+
+<!-- Include as many relevant details as possible about the environment you experienced the bug in -->
+
+- CPython versions tested on:
+- Operating system and version:
```

Comment:
Architecture?

Reply:
Good point, I think that's something I forgot to change from the mypy issue template!